lib/index.js

```diff
@@ -107,7 +107,7 @@ function renderFile (root, obj) {
 }
 
 function renderUrl (obj) {
-  return rest(obj.url).then((res) => { return JSON.parse(res.entity) })
+  return rest(obj.url).then((res) => { return JSON.parse(transformRaw(obj, res.entity)) })
```

Comment:
Hmm, yeah I feel like it couldn't hurt to do so. I can see a situation where someone might want to run a transform before the json parse in any of those situations. To abstract it out a little more, I might just add it in one place though. Here's what I thought when I looked at the code:

- Remove the json.parse from the direct `renderX` methods
- Add back the json.parse a single time in the [run function](https://github.com/static-dev/spike-records/blob/master/lib/index.js#L70), but only run the parse if the data is a string
- Before that json.parse run the transformRaw function once 😁 - this way it wouldnt even need to be abstracted as a function, could just run the ternary inline.

Does that make sense?